src/buildCommon.js

```diff
@@ -308,10 +424,110 @@ var spacingFunctions = {
     }
 };
 
+var greekCapitals = [
+    "\\Gamma",
+    "\\Delta",
+    "\\Theta",
+    "\\Lambda",
+    "\\Xi",
+    "\\Pi",
+    "\\Sigma",
+    "\\Upsilon",
+    "\\Phi",
+    "\\Psi",
+    "\\Omega"
+];
+
+/**
+ * Maps TeX font commands to objects containing:
+ * - variant: string used for "mathvariant" attribut in buildMathML.js
+ * - test: function acception mathord/textord value, returns true if the
+ *      variant or non-default font should be used.
+ */
+// A map between tex font commands an MathML mathvariant attribute values
+var fontMap = {
+    // styles
+    "mathbf": {
+        variant: "bold",
+        test: function(value) {
+            return /[a-zA-Z0-9]/.test(value.charAt(0));
+        }
+    },
+    "mathit": {
+        variant: "italic",
+        // All mathords and textords should be typeset using italics in \mathit.
+        // The test function has a special meaning in this case.  It determines
+        // which italics font to use when building an HTML layout and wheter or
+        // not to set mathvariant="italic" for MathML.
+        test: function(value) {
+            return /[0-9]/.test(value.charAt(0)) ||
+                utils.contains(["\\imath", "\\jmath"], value) ||
+                utils.contains(greekCapitals, value);
+        }
+    },
+    "mathrm": {
+        variant: "normal",
+        test: function(value) {
+            return /[a-zA-Z0-9]/.test(value.charAt(0));
+        }
+    },
+
+    // families
+    "mathbb": {
+        variant: "double-struck",
+        test: function(value) {
+            return/[A-Zk]/.test(value.charAt(0));
+        }
+    },
+    "mathcal": {
+        variant: "script",
+        test: function(value) {
+            return /[A-Z]/.test(value.charAt(0));
+        }
+    },
+    "mathfrak": {
+        variant: "fraktur",
+        test: function(value) {
+            return /[a-zA-Z0-9]/.test(value.charAt(0));
+        }
+    },
+    "mathscr": {
+        variant: "script",
+        test: function(value) {
+            return /[A-Z]/.test(value.charAt(0));
+        }
+    },
+    "mathsf": {
+        variant: "sans-serif",
+        test: function(value) {
+            // TODO(kevinb) update this when we implement unicode
+            return /[a-zA-Z]/.test(value.charAt(0)) ||
+                utils.contains(greekCapitals, value);
+        }
+    },
+    "mathtt": {
+        variant: "monospace",
+        test: function(value) {
+            // TODO(kevinb) update this when we implement unicode
+            return /[a-zA-Z0-9]/.test(value.charAt(0)) ||
+                utils.contains(greekCapitals, value);
+        }
+    }
+};
+
 module.exports = {
     makeSymbol: makeSymbol,
+    fontMap: fontMap,
     mathit: mathit,
     mathrm: mathrm,
+    mathbf: mathbf,
+    mathbb: mathbb,
+    mathcal: mathcal,
+    mathfrak: mathfrak,
+    mathtt: mathtt,
+    mathscr: mathscr,
+    mathsf: mathsf,
+    mathsym: mathsym,
```

Comment:
```mathit```, ```mathrm```, etc. are not common b/c they're only used in buildHTML.js and should be moved into that file.